### PR TITLE
Add Goal of AMSxTech Conference

### PR DESCRIPTION
**Pull request by Juice10**
Any comments are appreciated.
I added "one or two day conference format" but of course this could also be a three day 'un-conference' format if we want.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 AMSxTech Conference
 ===================
 
+## Goal
+
+AMSxTech has always been about crossover between professions in the technology world (design, development, pm, business, marketing, etc) and allowing for people to exchange knowledge and create connections they otherwise wouldn't be able to inside their own 'silo'. 
+Our goal is to do this in a one or two day conference format.
```
Comment (yopefonic) [inline README.md]: Is this line not more part of the "what" instead of the goal or "why"?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 AMSxTech Conference
 ===================
 
+## Goal
+
+AMSxTech has always been about crossover between professions in the technology world (design, development, pm, business, marketing, etc) and allowing for people to exchange knowledge and create connections they otherwise wouldn't be able to inside their own 'silo'.
```
Comment (yopefonic) [inline README.md]: Is connecting people across silo's more of a "How" and that we see startups as multi-discipline that come together with a goal more part of the "Why"?

**Comment by yopefonic** [comment]
@Juice10 I've added them manually and broke them up into the two sections I think they belong to.

**Comment by Juice10** [comment]
Sweet! Thanks @yopefonic!

**Pull request closed without merging**